test/lib/utils/issues.test.ts

```diff
@@ -84,4 +84,56 @@ describe('calculateTimeToFirstResponse', () => {
     const result = calculateTimeToFirstResponse(issues);
     expect(result).toBeCloseTo(0.0, 1);
   });
+});
+
+describe('calculateOpenIssuesRatio', () => {
+  test('should return undefined when openIssuesCount is undefined', () => {
+    expect(calculateOpenIssuesRatio(undefined, 100)).toBe(100.0);
+  });
+
+  test('should return undefined when totalIssuesCount is undefined', () => {
+    expect(calculateOpenIssuesRatio(10, undefined)).toBe(100.0);
+  });
+
+  test('should return undefined when both counts are undefined', () => {
+    expect(calculateOpenIssuesRatio(undefined, undefined)).toBe(100.0);
+  });
+
+  test('should return undefined when totalIssuesCount is 0', () => {
+    expect(calculateOpenIssuesRatio(0, 0)).toBe(100.0);
+  });
+
+  test('should calculate ratio correctly for 10 open out of 100 total', () => {
```

Comment:
these are all useless tests. just need 1 test that calculates a non-integer ratio, like 27/49